### DISCUSSION TITLE
Baram still active in developpment?

### DISCUSSION THREAD

**Discussion by torodecamargue** [nextfoam/baram#144] [General]
Hello Baram team,

Since several weeks there is no new messages from the Baram team. Is something is changed in Baram activity ? 

Best regards

**Comment by kdg5424** [comment]
Hello Julien.This is Donggyu.

Thank you for your concern. But we're doing our work very actively.
We've re-modeled our BARAM page. Now english manual is available.
And we're making BARAM tutorial videos and uploading on our youtube.
You can see it on our page and youtube.

Best regards

Donggyu Kim

**Reply by SergioP1975** [reply]
Thanks for all your efforts, BARAM is a very good software, the interface is very simple and usefull for beginners working in Windows.

**Comment by torodecamargue** [comment]
Thank you very much, Baram is a very good software ! And you make it better again
